src/main/java/com/twilio/rest/video/v1/Composition.java

```diff
@@ -234,8 +234,8 @@ private Composition(@JsonProperty("account_sid")
         this.accountSid = accountSid;
         this.status = status;
         this.dateCreated = DateConverter.iso8601DateTimeFromString(dateCreated);
-        this.dateCompleted = dateCompleted;
-        this.dateDeleted = dateDeleted;
+        this.dateCompleted = DateConverter.iso8601DateTimeFromString(dateCompleted);
+        this.dateDeleted = DateConverter.iso8601DateTimeFromString(dateDeleted);
```

Comment:
Need matching DateConverter.dateStringFromDateTime serializer function (only have DateConverter.dateStringFromLocalDate)

Reply:
The type on the input parameter changed when it shouldn't have. Working on a fix.

Reply:
Updated.